README.md

```diff
@@ -1 +1,64 @@
-# support-page
\ No newline at end of file
+# Support Page
+
+Use the Support Page Library with the GOV.UK Laravel library to insert a Support Page and Support Admin Page.
+
+## What's in the box?
+
+* Laravel 10 Blade GOV.UK Design support page and support page admin
+* Laravel 10 Blade GOV.UK Design support page components
+* PHP 8.2
+
+## Installation
+
+Via Composer: `composer require networkrailbusinesssystems/support-page`
+
+Publish via the terminal: `php artisan vendor:publish --provider="NetworkRailBusinessSystems\SupportPage\Providers\SupportPageProvider"`
+
+## Configuration
+
+* Add `'manage_support_page'` with admin rights to the UpdatePermissions Command
+* Add a link to manage Support Details on the admin blade with the route `support-page.admin.index` and wrap it in `@can'manage_support_page'`
+* Add `Route::supportPage();` to web.php
+
+* Register the form and import the class in the Govuk config:
+```php
```

Comment:
This code block is broken, but shouldn't duplicate existing documentation anyway.

If someone needs to know how to add a form to GOV.UK, they should refer to the GOV.UK documentation so that it is always up to date.